site/content/tutorial/13-advanced-styling/03-inline-styles/text.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Inline styles and the style directive
+---
+
+Apart from adding styles inside style tags, you can also add styles to individual elements using the style attribute. Usually you will want to do styling through CSS, but this can come in handy for dynamic styles, especially when combined with CSS custom properties.
+
+Add the following style attribute to the paragraph element:
+`style="color: {color}; --o: {bgOpacity};"`
+
+Great, now you can style the paragraph using variables that change based on your input without having to make a class for every possible value.
+
+This can however get unwieldly if you have to write a rather long string, and missing any of the semicolons will make it invalid. Svelte provides a nicer way to write inline styles and that is to use the style directive like this:
+
+```html
+<p 
```

Comment:
shall we break this into 2 tutorials?

1. use inline `style` attribute
2. use `style:` directives

Reply:
Was initially worried about the lack of content but seems like there's enough to go around for 2 tutorials. Updated it.